src/features/AuthModal/AuthModal.tsx

```diff
@@ -0,0 +1,130 @@
+        if (emailError) setEmailError('');
+        setEmail(e.target.value);
+    };
+
+    const handleChangePass = (e: ChangeEvent<HTMLInputElement>) => {
+        if (passError) setPassError('');
+        setPassword(e.target.value);
+    };
+
+    const handleRegister = () => {
+        const emailCorrect = emailChecking(email, setEmailError);
+        const passCorrect = passChecking(password, setPassError);
+
+        if (emailCorrect && passCorrect) {
+            dispatch(signUp({ email, password }));
+            onClose();
+        }
+    };
+
+    const handleLogin = () => {
+        const emailCorrect = emailChecking(email, setEmailError);
+        const passCorrect = passChecking(password, setPassError);
+
+        if (emailCorrect && passCorrect) {
+            dispatch(signIn({ email, password }));
+            onClose();
+        }
+    };
+
+    return (
+        <ModalWrapper
+            open={open}
+            onClose={onClose}
+        >
+            <Box sx={styles.wrapper} data-testid="AuthModal-wrapper">
+                <Typography
+                    variant="h5"
+                    align="center"
+                    data-testid="AuthModal-title"
+                >
+                    {isRegister
+                        ? t('Registration')
+                        : t('Login')}
+                </Typography>
+                <Box sx={styles.form}>
+                    <TextField
+                        placeholder={t('Email')}
+                        type="email"
+                        value={email}
+                        onChange={handleChangeEmail}
+                        error={!!emailError}
+                        helperText={t(emailError) || ' '}
+                        data-testid="AuthModal-email"
+                        label={t('Email')}
+                    />
+                    <PassField
+                        value={password}
+                        onChange={handleChangePass}
+                        error={!!passError}
+                        helperText={t(passError)}
+                        testedComponent="AuthModal"
+                    />
+                </Box>
+                { isRegister
+                    ? (
+                        <Button data-testid="AuthModal-btn-register" onClick={handleRegister}>
+                            {t('Registration')}
+                        </Button>
+                    )
+                    : (
+                        <Button data-testid="AuthModal-btn-login" onClick={handleLogin}>
+                            {t('Login')}
+                        </Button>
+                    )}
+                <Box sx={styles.caption}>
+                    <Typography data-testid="AuthModal-caption" variant="caption">
+                        { isRegister
+                            ? t('Already have an account?')
+                            : t('Do not have an account yet?')}
```

Comment:
`{t(isRegister ? 'Already have an account?' : ''Do not have an account yet?')}`

Reply:
Fixed